lava/assetlist.json

```diff
@@ -0,0 +1,32 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "lava-testnet-1",
+    "assets": [
+      {
+        "description": "The native token of Lava",
+        "denom_units": [
+          {
+            "denom": "ulava",
+            "exponent": 0
+          },
+          {
+            "denom": "lava",
+            "exponent": 6
+          }
+        ],
+        "base": "ulava",
+        "name": "LAVA",
+        "display": "lava",
+        "symbol": "LAVA",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/lava/images/lava-icon.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/lava/images/lava-icon.svg"
```

Comment:
One of these images has a black background and the other is transparent. Which is preferred? Can we make one of them an alternative kept in the folder but not referenced here?